internal/handlers/sqlite/msg_explain.go

```diff
@@ -83,6 +83,10 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		qp.Filter = params.Filter
 	}
 
+	if h.EnableSortPushdown {
+		qp.Sort = params.Sort
+	}
```

Comment:
We should update the `reply` document it currently returns `"sortingPushdown", false,`

Reply:
Nice catch, I've tested the pushdown, but didn't check the explain. Fixed